src/calculateRentalCost.js

```diff
@@ -5,6 +5,20 @@
  */
 function calculateRentalCost(days) {
   // write code here
+
+  if (days < 3) {
+    return days * 40;
+  }
+
+  if (days >= 7) {
+    return days * 40 - 50;
+  }
+
+  if (days >= 3) {
+    return days * 40 - 20;
+  }
```

Comment:
The condition `if (days >= 3)` is redundant because it is already covered by the previous conditions. You can remove this condition to simplify the logic.